nucypher/policy/conditions/auth/evm.py

```diff
@@ -106,3 +132,113 @@ def authenticate(cls, data, signature, expected_address):
             raise cls.AuthenticationFailed(
                 f"Invalid EIP4361 signature; signature not valid for expected address, {expected_address}"
             )
+
+
+class EIP1271Auth(EvmAuth):
+    EIP1271_ABI = """[
```

Comment:
Not por this PR, but at some moment we should create a more general organization to (1) store ABIs and (2) instantiate `Contract` entities from them (see https://github.com/nucypher/nucypher/issues/3503)